src/js/Head.js

```diff
@@ -167,4 +167,22 @@ var CindyJS = (function() {
                 console.log(JSON.stringify(state));
             };
 
+            CindyJS.debugState = function(index) {
```

Comment:
Interesting! Perhaps we should collect how to use these hidden debugging features somewhere.